.github/workflows/test.yaml

```diff
@@ -66,23 +110,27 @@ jobs:
           # `python_requires` from `setup.cfg`.
           - python-version: '3.7.1'
             tox-env: nodeps-withcov-posix
+            job-name: 'nodeps-test'
           # On PYPY coverage is very slow (5min vs 30min) as there is no C
           # extension.
           # There is very little PYPY specific code so there is not much to
           # gain from reporting coverage.
           - python-version: pypy-3.7
             tox-env: alldeps-nocov-posix
+            job-name: 'pypy-no-coverage'
+            skip-coverage: yes
           # We still run some tests with coverage,
           # as there are test with specific code branches for pypy.
           - python-version: pypy-3.7
-            tox-env: alldeps-withcov-posix
             trial-target: twisted.test.test_compat twisted.test.test_defer twisted.internet.test.test_socket twisted.trial.test.test_tests
+            job-name: 'pypy-coverage'
```

Comment:
We get some redundancy in the name from this -  eg "pypy-3.7-alldeps-nocov-posix".  Maybe `'coverage'` would be better?

Reply:
yes. I went for explicit 'with-coverage'